addons/account/models/account_account.py

```diff
@@ -703,6 +703,9 @@ def create(self, vals_list):
         return super().create(vals_list)
 
     def write(self, vals):
+        if 'prefix' in vals:
+            vals = {k: v for k, v in vals.items() if k not in ['code_digits', 'prefix']}
```

Comment:
del vals['prefix']
del vals['code_digits']

Reply:
but I think it's better to override _load_records_write for the account model and pop the these values there

also you can add test where you just call :
 self.env['account.account']._load_records([ 
     { 
         'xml_id': f"account.{company.id}_{xml_id}", 
         'values': values, 
         'noupdate': True, 
     }